polaris-react/src/components/Banner/Banner.stories.tsx

```diff
@@ -299,8 +299,8 @@ export const All = {
         </Text>
         <AllBanners
           onDismiss={() => {}}
-          action={{content: 'Primary action'}}
-          secondaryAction={{content: 'Secondary action'}}
+          action={{content: 'Primary'}}
+          secondaryAction={{content: 'Secondary'}}
```

Comment:
Made this change so we can see some examples of buttons side by side instead of stacked on small screens